ts/rollup.aqt.config.js

```diff
@@ -0,0 +1,15 @@
+import resolve from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import { terser } from "rollup-plugin-terser";
+
+import process from "process";
+const production = process.env["COMPILATION_MODE"] === "opt";
+
+export default {
+    output: {
+        name: "globalThis",
+        extend: true,
+        format: "iife",
+    },
+    plugins: [resolve({ browser: true }), commonjs(), production && terser()],
```

Comment:
I wonder if it might be better to re-use the same rollup config, and just override the part we need, so we don't need to maintain an extra file?

```patch
diff --git a/ts/editor/BUILD.bazel b/ts/editor/BUILD.bazel
index 59658204a..56e4b1b9e 100644
--- a/ts/editor/BUILD.bazel
+++ b/ts/editor/BUILD.bazel
@@ -27,7 +27,13 @@ ts_library(

 rollup_bundle(
     name = "editor",
-    config_file = "//ts:rollup.aqt.config.js",
+    args = [
+        "--output.name",
+        "globalThis",
+        "--output.extend",
+        "true",
+    ],
+    config_file = "//ts:rollup.config.js",
     entry_point = "index.ts",
     format = "iife",
     link_workspace_root = True,
```

The protobuf references are superfluous in this case though, and it's not a big win. What do you think?

Reply:
I have not forgotten about this, I will revisit this once I split off the card displaying functionality from the reviewer for use in the previewer, and see how that works out with different rollup setups.